src/components/card/Card.style.js

```diff
@@ -0,0 +1,66 @@
+import styled from "styled-components";
+import { keyframes } from "styled-components";
+
+export const Container = styled.div`
+    padding: 15px;
+    border-radius: 10px;
+    text-align: center;
+    box-shadow: 7px 12px 15px rgba(0,0,0,0.2);
+    animation: scaledown .4s linear;
+    animation-fill-mode: forwards;
+
+    :hover {
+        cursor: pointer;
+        z-index: 100;
+        animation: ${props => props.hover === 'on' ? 'scale .4s linear' : ''};
+        animation-fill-mode: ${props => props.hover === 'on' ? 'forwards' : ''};
+    }
+    @keyframes scaledown{
+        0%{
+            transform: scale(1.1);
+        }
+        100%{
+            transform: scale(1.0);
+        }
+    };
+    @keyframes scale{
+        0%{
+            transform: scale(1);
+        }
+        100%{
+            transform: scale(1.1);
+            box-shadow:10px 10px 20px 10px rgba(0,0,0,0.2);
```

Comment:
Move those to a separate file: `src/styled/animations.js`. PS: Look at styled-components doc on how to take css and combine it here.